packages/editor-ui/src/components/MfaSetupModal.vue

```diff
@@ -362,8 +369,4 @@ export default defineComponent({
 .notice {
 	margin: 0;
 }
-
-.modalContent {
-	overflow: hidden;
```

Comment:
Out of curiosity, the original design had the school. Why are we removing it 🤔 ?

Reply:
What do you mean by "the school"? I had to remove this to make the modal responsive